rpmlb/cli.py

```diff
@@ -73,6 +73,22 @@
     '--copr-repo', '-C', metavar='COPR_REPO',
     help='Target Copr for copr builder.',
 )
+@click.option(
+    '--epel', metavar='VERSION', type=int, default=7,
```

Comment:
Right now, yes, `KojiBuilder` is the only builder that requires it. However, multiple builders might require specification of the OS version to use…

This might be also related to the [platform element](https://github.com/sclorg/rpm-list-builder/issues/48) in recipes – in effect, this is a very specialized platform choice switch.

Having said that, and since the platform element is in discussion, `koji-epel` would be indeed better for now. In the future, this information should be extracted from the target platform specification, however that would be specified.

TL;DR: I agree and will change to `koji-epel`. Once the platform specifiers are finished, this option should be replaced by them.